ods_ci/tests/Resources/Files/accelerator/qe_accelerator_profile_2.yaml

```diff
@@ -0,0 +1,17 @@
+# File to provision an Accelerator Profile instance in test:
+#   "Modify An Accelerator Profile Using "Accelerator Profiles" Administration UI"
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  name: qeacceleratorprofile2
+  namespace: redhat-ods-applications
+spec:
+  description: description example
+  displayName: qe_accelerator_profile_2
```

Comment:
```suggestion
  name: accelerator-profile-test-2
  namespace: redhat-ods-applications
spec:
  description: You can give a meaningful description here, but if it is obvious, just remove this field completely.
  displayName: Accelerator Profile Test 2
```

Reply:
Not resolving it yet. Waitint for a reply above.